eng/pipelines/templates/jobs/run-cli-tests.yml

```diff
@@ -0,0 +1,87 @@
+resources:
+  repositories:
+    - repository: AzureCLI
+      type: github
+      name: azure/azure-cli
+      endpoint: Azure
+      ref: refs/heads/dev
+
+parameters:
+  - name: TargetRepoPackages
+    type: object
+    default: 
+      - 'sdk/core/azure-core'
+    # a list of any resolvable pip install. EG:
+    # - https://<path-to-blobstorage>/blah.whl
+    # - <packagename>==<version>
+  - name: InjectedPackages
+    type: object
+    default: []
+  - name: PythonVersion
+    type: string
+    default: "3.9"
+  - name: TargetModules
+    type: string
+    default: "appservice botservice cloud network azure-cli-core azure-cli-telemetry"
```

Comment:
This is the default list from [here](https://github.com/Azure/azure-cli/blob/cb3793acfabada7815fae83686d2b6324fa062a3/.azure-pipelines/templates/automation_test.yml#L43).